src/components/SingleDatePicker.jsx

```diff
@@ -344,7 +620,8 @@ export default class SingleDatePicker extends React.Component {
   }
 
   isHovered(day) {
-    return isSameDay(day, this.state.hoverDate);
+    const { hoverDate } = this.state || {};
```

Comment:
Hmm - yeah, i can see how that'd be unnecessarily weird. I guess this is fine as-is.